content/03.results.md

```diff
@@ -23,15 +23,15 @@ Along with the patient tumors, the Portal contains a small number of human tumor
 
 Each of the available samples contains summarized gene expression data from either single-cell or single-nuclei RNA sequencing.
 However, some samples also include additional data, such as quantified expression data from tagging cells with antibody-derived tags (ADT), such as CITE-seq antibodies [@doi:10.1038/nmeth.4380], or multiplexing samples with hashtag oligonucleotides (HTO) [@doi:10.1186/s13059-018-1603-1] prior to sequencing.
-Out of the over 700 samples, 95 have associated CITE-seq data, and 35 have associated multiplexing data.
+Out of the over 700 samples, 95 have associated CITE-seq data, and 35 have associated data with cell hashing.
 In some cases, multiple libraries from the same sample were collected for additional sequencing, either for bulk RNA-seq or spatial transcriptomics.
-Specifically, 182 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
+Specifically, 176 samples on the Portal were sequenced using bulk RNA-seq and 38 samples were sequenced using spatial transcriptomics.
 A summary of the number of samples with each additional modality is shown in Figure {@fig:fig1}B, and a detailed summary of the total samples with each sequencing method broken down by project is available in Table S1.
 
 Samples on the Portal are organized by project, where each project is a collection of similar samples from an individual lab.
 Users can filter projects based on diagnosis, included modalities (e.g., CITE-seq, bulk RNA-seq), 10x Genomics kit version (e.g., 10Xv2, 10Xv3), and whether or not a project includes samples derived from patient-derived xenografts or cell lines.
-The project card displays an abstract, the total number of samples included, a list of diagnoses for all samples included in the Project, and links to any external information associated with the project, such as publications and links to external data sources like SRA or GEO (Figure {@fig:fig1}C).
-The project card also indicates the type(s) of sequencing performed, including the 10x Genomics kit version, the suspension type (cell or nucleus), and if additional sequencing is present, like bulk RNA-seq or multiplexing.
+The project card displays an abstract, the total number of samples included, a list of diagnoses for all samples included in the Project, and links to any external information associated with the project, such as publications and links to external data, such as SRA or GEO (Figure {@fig:fig1}C).
+The project card also indicates the type(s) of sequencing performed, including the 10x Genomics kit version, the suspension type (cell or nucleus), and if additional sequencing is present, like bulk RNA-seq or cell hashing.
```

Comment:
```suggestion
The project card also indicates the type(s) of sequencing performed, including the 10x Genomics kit version, the suspension type (cell or nucleus), if additional sequencing is present, like bulk RNA-seq, or if the samples have been multiplexed using cell hashing.
```
Since we use "multiplexed" on the Portal and in the project card Figure, what about this?